modules/geo-layers/src/tile-layer/tileset-2d.js

```diff
@@ -97,13 +97,14 @@ export default class Tileset2D {
    * @param {*} viewport
    * @param {*} onUpdate
    */
-  update(viewport) {
+  update(viewport, {zRange}) {
```

Comment:
```suggestion
  update(viewport, {zRange} = {}) {
```

This is breaking unit tests.